.teamcity/common/extensions.kt

```diff
@@ -60,11 +60,11 @@ fun VcsSettings.filterDefaultBranch() {
 
 fun BuildType.applyDefaultSettings(os: Os = Os.linux, timeout: Int = 30, vcsRoot: String = "Gradle_Branches_GradlePersonalBranches") {
     artifactRules = """
-        build/report-* => .
-        buildSrc/build/report-* => .
-        subprojects/*/build/tmp/test files/** => test-files
-        subprojects/*/build/tmp/test files/** => test-files
-        build/errorLogs/** => errorLogs
+        build/report-* => .teamcity/
```

Comment:
In this way, how could we get the test files when we want to debug issues?

Reply:
ok, I see the "Hidden artifact" section in TeamCity documentation.

Reply:
Exactly. IIUC, guest users can't see the hidden artifacts.See https://teamcity-support.jetbrains.com/hc/en-us/community/posts/206890335-Downloading-hidden-artifacts-as-guest-user.

I didn't find any more documentation on the access rights to hidden artifacts.

What I just found is that we need to adjust the cleanup patterns to include hidden artifacts, if not they won't be cleaned up.